src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -210,3 +218,16 @@ def generate_document_context(contract_definition=None, user=None, order=None):
     }
 
     return apply_contract_definition_context_processors(context)
+
+
+def embed_images_in_context(context):
+    """Embed images in the context."""
+    edited_context = deepcopy(context)
+    try:
+        logo = DocumentImage.objects.get(id=edited_context["organization"]["logo_id"])
+        edited_context["organization"]["logo"] = image_to_base64(logo.file)
+    except DocumentImage.DoesNotExist:
+        edited_context["organization"]["logo"] = ORGANIZATION_FALLBACK_LOGO
```

Comment:
I don't think we want display a placeholder in production when the organization has no logo.

Reply:
This is only a one pixel image, to prevent error on document generation I suppose.
I believe that the same behavior existed before this PR.

Reply:
If the behavior was the same before then ok